packages/server/src/app/auth/services/auth.service.js

```diff
@@ -40,4 +42,38 @@ export class AuthService {
     }
     return user;
   }
+
+  /**
+   * @param username {string}
+   * @return {Promise<void>}
+   */
+  async requestPasswordReset(username) {
+    try {
+      const user = await UserService.instance().getByUsername(username);
+
+      // Generate a JWT that is valid for 10 minutes.
+      const jwtToken = JwtService.instance().encodeToken(user.toJSON(), 10 * 60 * 1000);
```

Comment:
I think 10 minutes is to short because sending a email may take some time. I suggest one hour or 30 minutes